ext/action-popup.html

```diff
@@ -11,94 +11,64 @@
     <link rel="icon" type="image/png" href="/images/icon48.png" sizes="48x48">
     <link rel="icon" type="image/png" href="/images/icon64.png" sizes="64x64">
     <link rel="icon" type="image/png" href="/images/icon128.png" sizes="128x128">
+    <link rel="stylesheet" type="text/css" href="/css/material.css">
     <link rel="stylesheet" type="text/css" href="/css/action-popup.css">
     <script src="/js/pages/action-popup-main.js" type="module"></script>
 </head>
 <body>
-
-<div id="loading">
```

Comment:
Why is this removed?

Reply:
Removing the loading message is an absolute no go. This needs to be brought back.